backend/modules/pdf_processing/ai_extraction_service.py

```
@@ -261,49 +167,67 @@ async def extract_personas_and_key_figures(
 
 CRITICAL: You must identify ALL named individuals, companies, organizations, and significant unnamed roles mentioned within the case study narrative.
 
-Instructions for key_figures identification:
-- Find ALL types of key figures that can be turned into personas
-- Include both named and unnamed entities that are part of the business story
-- Even if someone/thing is mentioned only once or briefly, include them if they have a discernible role
+━━━ KEY FIGURES IDENTIFICATION ━━━
+- Find ALL named individuals and significant roles — aim for at least 4–6 personas
```

Ruff (0.15.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)